app/Http/Controllers/Admin/OrderListController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Services\Customer\OrderService;
+use Illuminate\Http\Request;
+
+class OrderListController extends Controller
+{
+    private $orderService;
+
+    /**
+     * @param $customerService
+     */
+    public function __construct(OrderService $orderService)
+    {
+        $this->orderService = $orderService;
+    }
+
+    public function index()
+    {
+        $orders = $this->orderService->getListOrder();
+        //If get date ... to date ... then filter products by date
+        if (request()->date_from && request()->date_to) {
+            $orders = $this->orderService->getListOrderByDate(request()->date_from, request()->date_to);
+        }
+        return view('admin.order.listorder', ['orders' => $orders]);
+    }
+
+    public function store(Request $request)
```

Comment:
why this name?

Reply:
I fixed it, I'm so sorry for not distinguishing between store and search